### ISSUE TITLE
IfcSweptDiskSolid, rebar, and AbRV

### ISSUE THREAD

**Issue by JOuellette-Autodesk** [buildingSMART/IFC4.x-IF#94]
**Problem**
IfcSweptDiskSolid is currently out of scope for AbRV, but in scope for RV. It should be in scope for AbRV to accommodate optimal representation of concrete reinforcement (aka "rebar")for infrastructure elements like reinforced concrete for bridge elements, tunnels, etc.

**Solution(s)**
Include IfcSweptDiskSolid in the AbRV.

**Require schema changes?**
- ` ` yes
- `✓` no
- ` ` don't know

**Require documentation changes?**
- `✓` yes
- ` ` no
- ` ` don't know

**Rule required**
*Need for a formal rule? Describe it*
- ` ` normative check: every IFC file must pass this check
- ` ` good practice check: just triggers a warning, but the IFC file is still valid
- ` ` don't know

A clear and concise description of the rule 

**Additional context**

**Comment by aothms** [comment]
I agree AbRV should be a strict superset of RV in terms of concepts.

I would personally also be ok with the other solution to remove swept-disk from RV. As RV should be about an explicit, unambiguous, universally understood (and hopefully foolproof) representation of geometry. swept-disk sits a bit in the middle there. And then maybe reintroduce something like we had with CV if you care about efficient/idiomatic/semantically richer geometry descriptions. But that's another discussion.

I would appreciate if somebody could take the time to unify the complete set of discrepancies between RV and AbRV in terms of templates. E.g in addition to this, what about the non-alignment concepts that AbRV selects, but RV doesn't?

**Comment by RickBrice** [comment]
Just to add to the conversation - IfcSectionedSolidHorizontal (8.8.3.35.1) seems to be excluded from AbRV and yet its purpose seems to be directly related to alignments and bridges.

**Comment by aothms** [comment]
Indeed. There is a template for it, but it's not included in any mvd. https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/content.html
